### PR TITLE
Delete the card body in list view

### PR DESCRIPTION
**Pull request by 12Chao**
The change about card body were made in template/base.html, and the other changes were made for ace editor which did not work.

### CODE REVIEW

File: ChemCheck/upload/static_app/JS/aceeditor.js [hunk 1]
```diff
@@ -12,4 +20,4 @@ $(document).ready(function(){
         editor.setValue($("#file-content").text());
         editor.setOptions({
         maxLines: 2000
-        });
\ No newline at end of file
+        });}
```
Comment (bryanwweber) [inline ChemCheck/upload/static_app/JS/aceeditor.js]: You should add a blank newline at the end of this file. Also make sure that `db.sqlite3` is added to `.gitignore`

**Review by bryanwweber — APPROVED** [review]

File: ChemCheck/upload/static_app/JS/aceeditor.js (hunk 1 above)
Comment (12Chao) [reply ChemCheck/upload/static_app/JS/aceeditor.js]: Thanks Bryan. I just changed them.

File: ChemCheck/upload/static_app/JS/aceeditor.js [hunk 2]
```diff
@@ -20,4 +20,6 @@ $(document).ready(function(){
         editor.setValue($("#file-content").text());
         editor.setOptions({
         maxLines: 2000
-        });}
\ No newline at end of file
+        });}
+
```
Comment (bryanwweber) [inline ChemCheck/upload/static_app/JS/aceeditor.js]: Just one line here :smiley:

Comment (12Chao) [reply ChemCheck/upload/static_app/JS/aceeditor.js]: Sorry, I just edited it again 😄

**Comment by rwest** [comment]
You're getting the hang of things. The first commit is called "delete the card body in list view" but it also edits the javascript for the ajax and ace editor stuff.  Remember to try to keep the changes within a commit related to one thing, that can be easily described by the commit message. A GUI for your git commits (you can use VSCode) will help you see what changes your'e about to commit and stage just some of them -- you don't need to commit all your changes at once, even if you've done more than one thing since you last committed.

**Comment by rwest** [comment]
Also, this doesn't include the most recent changes from Master. You could rebase, or hope nothing breaks when you merge (it's unlikely too from what I can see)

**Comment by 12Chao** [comment]
OK, I will discard this one make a new one by git GUI.

**Comment by rwest** [comment]
No need. Just something to think about moving forward.
It's OK to have a bit of a messy history, especially near the start.

**Comment by rwest** [comment]
great. Now before doing any more commits on your `chaowork` branch, be sure to pull in the master to it (eg. `git pull origin master`)

**Comment by 12Chao** [comment]
OK, Thanks for the suggestions, I will make it better next time 😅

**Pull request merged**